src/components/common/Icon/Icon.js

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 const Icon = ({ icon, iconColor, size }) => (
     <i className="material-icons" style={{color: iconColor, fontSize: size}}>
         {icon}
     </i>
 );
 
+Icon.propTypes = {
+    icon: PropTypes.string,
+    color: PropTypes.string,
+    size: PropTypes.number
```

Comment:
```suggestion
    size: PropTypes.string
```
[exemplo](https://github.com/OpenDevUFCG/glossario-ufcg/blob/3f468202b017983dc5bf593719ef4ca9911cc335/src/components/common/Icon/IconWithText/IconWithText.js#L8)